streams/src/main/java/org/apache/kafka/streams/TopologyDescription.java

```diff
@@ -57,7 +57,7 @@
     /**
      * Represents a {@link Topology#addGlobalStore(org.apache.kafka.streams.processor.StateStoreSupplier, String,
      * org.apache.kafka.common.serialization.Deserializer, org.apache.kafka.common.serialization.Deserializer, String,
-     * String, ProcessorSupplier)} global store}.
+     * String, org.apache.kafka.streams.processor.ProcessorSupplier)} global store}.
```

Comment:
Are you sure this is correct? It's too different packages and thus the full package name in required IMHO.